lib/content-services/src/lib/common/services/nodes-api.service.ts

```diff
@@ -239,4 +239,26 @@ export class NodesApiService {
 
         return new NodeMetadata(metadata, nodeEntry.entry.nodeType);
     }
+
+    /**
+     * Gets the list of holds assigned to the node.
+     *
+     * @param nodeId ID of the target node
+     * @returns List of assigned holds
+     */
+    getNodeAssignHolds(nodeId: string): Observable<Hold[]> {
```

Comment:
This is public API so it should support the same options as `listParents` e.g. pagination and node includes query options

Reply:
added